apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -82,6 +88,14 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 .commit();
     }
 
+    public static String defaultLogUrlFunction(String startType){
```

Comment:
Don't name a function "function"

```suggestion
    public static String generateLogUrl(String startType) {
```

Reply:
Yes, resolved in new commit.